cmd/kubeadm/app/util/version_test.go

```diff
@@ -121,9 +121,10 @@ func TestVersionFromNetwork(t *testing.T) {
 		"stable-1.3": {"v1.3.10", "v1.3.10", false, false},
 		"latest":     {"v1.6.0-alpha.0", "v1.6.0-alpha.0", false, false},
 		"latest-1.3": {"v1.3.11-beta.0", "v1.3.11-beta.0", false, false},
-		"empty":      {"", currentVersion, true, false},
-		"garbage":    {"<?xml version='1.0'?><Error><Code>NoSuchKey</Code><Message>The specified key does not exist.</Message></Error>", currentVersion, true, false},
-		"unknown":    {"The requested URL was not found on this server.", currentVersion, true, false},
+		"empty":      {"", "", true, true},
+		"garbage":    {"", "", true, true},
+		"unknown":    {"", "", true, true},
```

Comment:
delete my previous comments, as they were wrong.
please format this like so and the PR is good to go:

```
	cases := map[string]T{
		"stable":          {"stable-1", "v1.4.6", false, false}, // recursive pointer to stable-1
		"stable-1":        {"v1.4.6", "v1.4.6", false, false},
		"stable-1.3":      {"v1.3.10", "v1.3.10", false, false},
		"latest":          {"v1.6.0-alpha.0", "v1.6.0-alpha.0", false, false},
		"latest-1.3":      {"v1.3.11-beta.0", "v1.3.11-beta.0", false, false},
		"latest-1.5":      {"", currentVersion, true, false}, // fallback to currentVersion on fetcher error
		"invalid-version": {"", "", false, true},             // invalid version cannot be parsed
	}
```